treeherder/config/settings.py

```diff
@@ -397,17 +397,17 @@
         }
     },
     'daily-intermittents-commenter': {
-        # Executes every morning at 7 a.m. UTC
+        # Executes every morning at 7 a.m. UTC except sunday
         'task': 'intermittents-commenter',
-        'schedule': crontab(minute=0, hour=7),
+        'schedule': crontab(minute=0, hour=7, day_of_week='monday-saturday'),
```

Comment:
The Celery docs aren't clear what form of the name for the days can be used (their one example uses the short form, "mon-fri") - so I checked and this works fine:

```python
>>> from celery.schedules import crontab_parser
>>> crontab_parser(7).parse('monday-saturday')
set([1, 2, 3, 4, 5, 6])
```